hibernate-spatial/src/main/java/org/hibernate/spatial/predicate/GeolatteFilterPredicate.java

```diff
@@ -31,15 +20,15 @@
 /**
  * {@link JTSFilterPredicate}, but for geolatte-geom.
  */
-public class GeolatteFilterPredicate extends AbstractSimplePredicate implements Serializable {
+public class GeolatteFilterPredicate {
```

Comment:
Obsolete class?

Reply:
No, but I haven't yet updated this code to 6. I will add a todo tag for this